src/javascript/JContent/ContentRoute/ContentLayout/FilesGrid/FileCard/FileCard.jsx

```diff
@@ -185,6 +195,14 @@ export const FileCard = ({
                             {`${node.width.value} x ${node.height.value}`} <FileSize node={node}/>
                         </Typography>
                     </div>}
+                    {showSubNodes &&
```

Comment:
See showSubNodes comment above. Need to make sure it's a boolean and not just falsey value.

Reply:
Yup I believe with switching to optional chaining it will be a boolean value or null/undefined (in both cases it should be OK)